Templates/Cartesian.swift.gyb

```diff
@@ -0,0 +1,72 @@
+//
+//  Cartesian.swift.gyb
+//  SwiftCheck
+//
+//  Created by Adam Kuipers on 5/10/16.
+//  Copyright © 2016 Typelift. All rights reserved.
+//
+
+%{
+MAX_ARITY = 22
+}%
+
+extension Gen /*: Cartesian*/ {
```

Comment:
Could you add a comment here or in the file header that this file is machine generated and should not be edited?